auth.md

```diff
@@ -0,0 +1,39 @@
+# Auth
+
+By default, we're using cookies as a mock auth method. It's for trying out OpenGPTs.
+For production, we recommend using JWT auth, outlined below.
+
+## JWT Auth: Options
+
+There are two ways to use JWT: Basic and OIDC. The main difference is in how the key
+used to decode the JWT is obtained. For the Basic method, you'll provide the decode
+key as a Base64-encoded string in an environment variable. For the OIDC method, the
+key is obtained from the OIDC provider automatically.
+
+### JWT Basic
+
+To use JWT Basic, set `AUTH_TYPE=jwt_basic`. Then, set the issuer, audience, and
+the decode key in Base64 format. Audience can be one or many - just separate them
+with commas.
+
+```bash
+export AUTH_TYPE=jwt_basic
+export JWT_DECODE_KEY_B64=<base64_decode_key>
+export JWT_ISS=<issuer>
+export JWT_AUD=<audience>  # or <audience1>,<audience2>,...
+```
+
+Base64 is used for the decode key because handling multiline strings in environment
+variables is error-prone. Base64 makes it a one-liner, easy to paste in and use.
+
+### JWT OIDC
```

Comment:
I would pump this before the JWT Basic and maybe call JWT Basic something like jwt_local. JWT Basic sounds to me like "preferred" or "default" - I would avoid that view.

Reply:
Sounds reasonable. Will change.